src/System.Net.Http.WinHttpHandler/src/project.json

```diff
@@ -1,6 +1,7 @@
 {
   "dependencies": {
     "Microsoft.Win32.Primitives": "4.0.0",
+    "System.Diagnostics.DiagnosticSource": "4.0.0-rc3-23729",
```

Comment:
Yes.  We update them en mass to move all values in the repo to the next known-good set of packages.
https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/project-nuget-dependencies.md#upgrading-a-package-dependency